README.md

```diff
@@ -1,10 +1,45 @@
+[![Build Status](https://badge.buildkite.com/9e0e6c88972a3248a0908506d6946624da84e4e18c0870c4d0.svg)](https://buildkite.com/rust-vmm/kvm-ioctls-ci)
+![crates.io](https://img.shields.io/crates/v/$CRATE.svg)
+
 # kvm-ioctls
 
-TODO
+The kvm-ioctls crate provides safe wrappers over the KVM API, a set of ioctls
```

Comment:
A link to https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt would be nice.